src/main/java/frc/robot/subsystems/intake/IntakeConstants.java

```diff
@@ -0,0 +1,23 @@
+package frc.robot.subsystems.intake;
+
+import static edu.wpi.first.units.Units.Degrees;
+
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+
+public class IntakeConstants {
+    public static final double GEAR_RATIO = 0;
+    public static final double ANGLE_KP = 0;
+    public static final double ANGLE_KI = 0;
+    public static final double ANGLE_KD = 0;
```

Comment:
Should be of type `LoggedTunableNumber`.

Reply:
done.

Reply:
87e7e5cac0b80010fdf2f71007ae5eb93d563786